### PR TITLE
Expose winkernel settings as flags instead of hard-coded values...

### PR DESCRIPTION
**Pull request by marosset**
Signed-off-by: Mark Rossetti <marosset@microsoft.com>

### CODE REVIEW

**Comment by marosset** [comment]
/hold cancel
This is working (for me ☺)

File: backends/windows/kernelspace/sink.go [hunk 1]
```diff
@@ -141,20 +158,30 @@ func (s *Backend) Setup() {
 		15*time.Second,
 		"sync period duration")
 
-	klog.V(0).InfoS("Using Windows Kernel Proxier. node ip  %v", nodeip)
+	klog.Info("Starting Windows Kernel Proxier.")
+	klog.InfoS("  Cluster CIDR", "clusterCIDR", *clusterCIDR)
```
Comment (knabben) [inline backends/windows/kernelspace/sink.go:162]: should this be on (5) or debug level?

File: backends/windows/kernelspace/proxier.go [hunk 2]
```diff
@@ -277,12 +277,12 @@ func NewProxier(
 	masqueradeMark := fmt.Sprintf("%#08x/%#08x", masqueradeValue, masqueradeValue)
 
 	if nodeIP == nil {
-		klog.InfoS("Invalid nodeIP, initializing kube-proxy with 10.20.30.11 as nodeIP")
+		klog.Warning("Invalid nodeIP, initializing kube-proxy with 10.20.30.11 as nodeIP")
```
[the remark this answers is not in the document]
Comment (marosset) [reply backends/windows/kernelspace/proxier.go:280]: yea probably. we're defaulting --node-ip in `sink.go` and the value gets logged there.

File: backends/windows/kernelspace/sink.go (hunk 1 above)
Comment (marosset) [reply backends/windows/kernelspace/sink.go:162]: I'm generally a fan of logging parameters at max verbosity but up to you

File: backends/windows/kernelspace/proxier.go [hunk 3]
```diff
@@ -920,12 +918,12 @@ func (proxier *Proxier) syncProxyRules() {
 	// Update service healthchecks.  The endpoints list might include services that are
 	// not "OnlyLocal", but the services list will not, and the serviceHealthServer
 	// will just drop those endpoints.
-//	if err := proxier.serviceHealthServer.SyncServices(serviceUpdateResult.HCServiceNodePorts); err != nil {
-//		klog.ErrorS(err, "Error syncing healthcheck services")
-//	}
-//	if err := proxier.serviceHealthServer.SyncEndpoints(endpointUpdateResult.HCEndpointsLocalIPSize); err != nil {
-//		klog.ErrorS(err, "Error syncing healthcheck endpoints")
-//	}
+	//	if err := proxier.serviceHealthServer.SyncServices(serviceUpdateResult.HCServiceNodePorts); err != nil {
```
[the remark this answers is not in the document]
Comment (knabben) [reply backends/windows/kernelspace/proxier.go:921]: I think we do, but it should be enabled for the other BE as well.

**Pull request merged**